cfdm/test/test_Field.py

```diff
@@ -609,6 +609,34 @@ def test_Field_compress_uncompress(self):
                     self.assertTrue(f.equals(c, verbose=3), message)
         # --- End: for
 
+    def test_Field_creation_commands(self):
+        if self.test_only and inspect.stack()[0][3] not in self.test_only:
+            return
+
+        f = self.f.copy()
+
+        for rd in (False, True):
+            for indent in (0, 4):
+                for s in (False, True):
+                    for ns in ('cfdm', ''):
+                        _ = f.creation_commands(representative_data=rd,
+                                                indent=indent,
+                                                namespace=ns,
+                                                string=s)
+        # --- End: for
+
+        for i in range(7):
```

Comment:
Should probably tidy this a little for readability if nothing else, by moving this `for i in` loop with the example field definition & inner `creation_commands` call under the four nested four loops above, to avoid repeating them.